lib/elastic_sync/index.ex

```diff
@@ -3,10 +3,10 @@ defmodule ElasticSync.Index do
   alias Tirexs.Resources.APIs, as: API
 
   # TODO: Allow developers to control mappings here
-  def create(names) do
+  def create(names, config \\ []) do
```

Comment:
I don't think we need to provide a default here. I prefer the "explicit style" in this case I think.